build/task-build.js

```diff
@@ -44,6 +27,9 @@ async function buildFile({ path: sourceFile, stats: sourceStats }, targetFile) {
 
   const extension = path.extname(sourceFile);
   if (extension === '.js' || extension === '.jsx') {
+    if (extension === '.jsx') {
+      targetFile = targetFile.substring(0, targetFile.length - 1);
```

Comment:
This looks a bit terrible. If by any chance we'd like to also support files with the extension .foo at some point, we'd have to substring -3 and add a .js etc.

Extracting the basename and patch and attaching an extension seems like a better way to do this, but I can totally understand why you'd not want to do it now.